book/tutorials/gpr/gpr.ipynb

```diff
@@ -0,0 +1,518 @@
+   "outputs": [],
+   "source": [
+    "# Collect all Point Data where the instrument string contains the GPR in its name\n",
+    "#qry = session.query(PointData).filter(PointData.instrument.contains('GPR'))\n",
+    "# Print all types of PointData in the query\n",
+    "tmp = session.query(PointData.instrument).distinct().all()\n",
+    "print(tmp)\n",
+    "# Print all types of PointData in the query\n",
+    "tmp = session.query(PointData.type).distinct().all()\n",
+    "print(tmp)\n",
+    "\n",
+    "qry = session.query(PointData).filter(PointData.type == 'two_way_travel')\n",
+    "\n",
+    "# Additionally Filter by a date \n",
+    "qry = qry.filter(PointData.date==datetime.date(2020, 1, 29))\n",
+    "\n",
+    "# See upload details at https://github.com/SnowEx/snowexsql/blob/087b382b8d5098f09db67310efd49f777525c0c8/scripts/upload/add_gpr.py#L27\n",
+    "\n",
+    "# Grab all the point data in the buffer using the POSTGIS ST_Within, anytime using the postgis functions we typically have to convert to text\n",
+    "qry = qry.filter(func.ST_Within(PointData.geom.ST_AsText(), buffered_pit.ST_AsText()))\n",
+    "\n",
+    "# Use our handy dandy function to execute the query and make it a geopandas dataframe\n",
+    "dfGPR = query_to_geopandas(qry, engine)\n",
+    "# rename 'value' in dataframe as 'twt'\n",
+    "dfGPR.rename(columns={'value': 'twt'},inplace=True )\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "peripheral-violation",
+   "metadata": {},
+   "source": [
+    "### Step 5: Plot it!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "precious-injection",
+   "metadata": {
+    "tags": [
+     "nbsphinx-gallery",
+     "nbsphinx-thumbnail"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Get the Matplotlib Axes object from the dataframe object, color points by snow depth value\n",
+    "ax = dfGPR.plot(column='twt', legend=True, cmap='PuBu')\n",
+    "\n",
+    "\n",
+    "# Use non-scientific notation for x and y ticks\n",
+    "ax.ticklabel_format(style='plain', useOffset=False)\n",
+    "\n",
+    "# Set the various plots x/y labels and title.\n",
+    "ax.set_title('Grand Mesa GPR Travel-times w/in {}m of site {}'.format(buffer_dist, site_id))\n",
+    "ax.set_xlabel('Easting [m]')\n",
+    "ax.set_ylabel('Northing [m]')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "english-underwear",
+   "metadata": {},
+   "source": [
+    "### Step 6: Convert TWT to Depth Using Snow Density\n",
+    "We will relate the dry snow density to the electromagnetic wave speed using the Kovacs et. al (1995) formula\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\epsilon_{\\mathrm{r}}^{\\prime}=(1+0.845 \\rho)^{2} \\quad .\n",
+    "\\end{equation}\n",
+    "\n",
+    "This equation calculates the dielectric constant $\\epsilon_{\\mathrm{r}}^{\\prime}$, provided the snow density $\\rho$. We must then relate the dielectric constant to the electromagnetic wave speed $(v)$\n",
```

Comment:
If you could please use $$ instead of \begin{equation}, this allows referencing equations by a label and will render correctly in both the  notebook and the jupyterbook website: 

```
$$
v = \frac{c}{\sqrt{\epsilon_{\mathrm{r}}^{\prime}}} \quad ,
$$ (wavespeed)

In {eq}`wavespeed`,  $c$ is the universal constant $0.3~m/ns$.
```

Reply:
Alternatively, looks like you can keep the \begin{equation} blocks if you add this line to the book configuration file https://github.com/snowex-hackweek/website/blob/410a452e03e8239a755de9d485714345e97e1519/book/_config.yml#L9 
```
parse:
  myst_enable_extensions:
    - amsmath
```

Reply:
oh shoot, well I just pushed you the changes using the $$ way. Let me know how it compiles on your end, I can make changes or use this suggestion to revert to this method.